util/winutil/winutil_windows.go

```diff
@@ -18,15 +22,21 @@ const (
 )
 
 // GetDesktopPID searches the PID of the process that's running the
-// currently active desktop and whether it was found.
+// currently active desktop or error.
 // Usually the PID will be for explorer.exe.
-func GetDesktopPID() (pid uint32, ok bool) {
+func GetDesktopPID() (pid uint32, err error) {
 	hwnd := windows.GetShellWindow()
 	if hwnd == 0 {
-		return 0, false
+		err = errors.New("no Shell process is present")
+		return
 	}
+
 	windows.GetWindowThreadProcessId(hwnd, &pid)
-	return pid, pid != 0
+	if pid == 0 {
+		err = fmt.Errorf("invalid PID for HWND %v", hwnd)
```

Comment:
Get into the habit of avoid a "naked return" unless your func is like 2 lines.

return 0, fmt.Errorf(...)
...

return pid, nil